player/src/main/java/com/focusit/jsflight/player/scenario/UserScenario.java

```diff
@@ -100,11 +104,33 @@ public void applyStep(int position)
 
             String target = getTargetForEvent(event);
 
-            element = SeleniumDriver.findTargetWebElement(event, target);
+            String type = event.getString("type");
 
-            SeleniumDriver.processMouseEvent(event, element);
+            SeleniumDriver.waitPageReady(event);
 
-            SeleniumDriver.processKeyboardEvent(event, element);
+            switch (type)
+            {
+            case "scroll":
```

Comment:
Dirty hack. Scroll event hasn't target you scrolled to. it has nothing in it's source (target) field. The best thing you can do is to find a real target to scroll to from other events. Or you can rename event-type to something like scroll-emulation.